examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  children: React.ReactNode;
+  title: string;
+};
+
+const getTabKey = (index: number, suffix?: string) =>
+  [`tab-${index}`, suffix].filter(Boolean).join('-');
+
+export function Tabs({ children }) {
+  const [currentTab, setCurrentTab] = useState(0);
+  const tabsRefs: HTMLElement[] = [];
+
+  const onKeyPress = (event: React.KeyboardEvent) => {
```

Comment:
Since this is called `onKeyDown` it'd make more sense.

```suggestion
  const onKeyDown = (event: React.KeyboardEvent) => {
```